package.json

```diff
@@ -12,9 +12,10 @@
     "devDependencies": {
         "@react-native-community/eslint-config": "^0.0.5",
         "eslint": "^6.5.1",
-        "react": "16.9.0",
-        "react-native": "0.61.5",
-        "react-native-windows": "^0.61.0-0"
+        "react": "18.0.0",
+        "react-native": "0.69.5",
+        "react-native-test-app": "^1.6.13",
```

Comment:
I don't think you need this react-native-test-app as devDependency here ?

Reply:
Removed. [a0cecca](https://github.com/react-native-video/react-native-video/pull/2845/commits/a0ceccaad8ff1a06c9b67757d3c312bddc1c38e0)

Reply:
> 63538df

Package names are now fixed [63538df](https://github.com/react-native-video/react-native-video/pull/2845/commits/63538df3f296bcefbe8c29247c9311459a04c89c).

Reply:
I think you did a mistake here, you removed react-native-test-app from exemple/basic/package.json but it is need here
And let it in root package.json where it is not needed.
(Then currently the sample doesn't build ... )

Reply:
Oops. I fixed it now.